rl_algorithms/common/abstract/agent.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, env: gym.Env, args: argparse.Namespace, log_cfg: ConfigDict):
                 f"./checkpoint/{self.env_name}/{log_cfg.agent}/{log_cfg.curr_time}/"
             )
             os.makedirs(self.ckpt_path, exist_ok=True)
+            self.log_cfg["ckpt_path"] = self.ckpt_path
```

Comment:
It can be changed `log_cfg["ckpt_path"] -> log_cfg.ckpt_path` because it is ConfigDict.

And I think it should place abstract learner.

Reply:
Fixed!